Tests/UnitTests/Mocks/MockNSWindow.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  MockNSWindow.swift
+//
+//  Created by Will Taylor on 2/25/25.
+
+import Foundation
+
+#if canImport(AppKit)
+import AppKit
+
+extension NSWindow {
+    static func mock() -> NSWindow? {
```

Comment:
I know I know, this isn't great, but reflection is the only way I could find to get access to a NSWindow/UIScene in unit tests 😅